src/main/java/com/phamhieu/exercise/basic/string/String02.java

```diff
@@ -0,0 +1,14 @@
+package com.phamhieu.exercise.basic.string;
+
+public class String02 {
+
+    //Kiểm tra chuỗi ký tự có đối xứng hay không. (Ví dụ abcdcba là đối xứng)
+    public boolean checkSymmetryString(final String input) {
+        for (int i = input.length() - 1; i >= input.length() / 2; i--) {
```

Comment:
Sẽ gọn hơn nếu tính for i=0 --> length / 2

Reply:
done